elytron/src/main/java/org/wildfly/extension/elytron/SSLDefinitions.java

```diff
@@ -719,9 +705,40 @@ protected void executeRuntimeStep(OperationContext context, ModelNode operation)
                                 ((ReloadableX509ExtendedTrustManager) trustManager).reload();
                             }
                         })
+                .addOperation(new SimpleOperationDefinitionBuilder(ElytronDescriptionConstants.INIT, RESOURCE_RESOLVER)
+                        .setRuntimeOnly()
+                        .build(), init("TrustManager"))
                 .build();
     }
 
+    private static OperationStepHandler init(String managerType) {
```

Comment:
I think something like ServiceUtil<?> may be able to be passed here instead of a string.